tests/integration/defs/test_e2e.py

```
@@ -1653,6 +1653,14 @@ def test_openai_responses(llm_root, llm_venv):
          str(test_root / "_test_openai_responses.py")])
 
 
+def test_openai_responses_entrypoint(llm_root, llm_venv):
```

Ruff (0.14.8)

Unused function argument: `llm_root`

(ARG001)